packages/api-gateway/src/drivers/http/server.js

```diff
@@ -8,7 +8,7 @@ async function start() {
 
 	const fastify = Fastify({ logger: true });
 
-	fastify.register(Autoload, { dir: path.join(__dirname, 'routes')});
+	fastify.register(Autoload, { dir: path.join(__dirname, 'routes/places')});
```

Comment:
@Alejandrocuartas We shouldn't modify this configuration because it will break the entire routing system.  As we have it, the autoload plugin loads the routes bases on the folders, so creating the `places` folder inside the routes folder is enough.